modules/common/src/main/scala/surge/health/config/WindowingStreamConfig.scala

```diff
@@ -51,9 +54,18 @@ object WindowingStreamConfigLoader {
     val maxStreamSize = config.getInt("surge.health.window.stream.max-size")
     val frequencies = config.getDurationList("surge.health.window.stream.frequencies").asScala.map(d => FiniteDuration(d.toMillis, "millis"))
 
+    val drainPerTick = Try { config.getInt("surge.health.stream.drain-per-tick") }.toOption.getOrElse(10)
+
+    val tickInterval =
+      Try { config.getDuration("surge.health.stream.tick-duration") }.toOption.map(d => FiniteDuration(d.toMillis, "millis")).getOrElse(250.millis)
+
+    val initialTickDelay =
+      Try { config.getDuration("surge.health.stream.initial-tick-delay") }.toOption.map(d => FiniteDuration(d.toMillis, "millis")).getOrElse(1.second)
```

Comment:
Same here with `FiniteDuration` - just do a `config.getDuration("surge.health.stream.initial-tick-delay").toMillis.millis` in case of stray typos

Reply:
done